client.go

```diff
@@ -484,7 +504,7 @@ func (c *APIClient) startQueryRequest(ctx context.Context, request *QueryRequest
 		respHeaders http.Header
 	)
 	err := c.doRetry(func() error {
-		return c.doRequest(ctx, "POST", path, request, &resp, &respHeaders)
+		return c.doRequest(ctx, "POST", path, request, c.NeedSticky(), &resp, &respHeaders)
```

Comment:
Only check sticky at post?

Reply:
polling query result always need to be sticky, so fixed to true here